### ISSUE TITLE
Warning maybe due to legacy plugin SDK

### ISSUE THREAD

**Issue by emeric-martineau** [wgebis/terraform-provider-mailgun#14]
Hello,

I currently add credential support on this terraform provider.

I see some warnings.

Maybe just need update terraform sdk version.

### Terraform Version
0.13.0, 0.14.0

### Affected Resource(s)
Routes, Credentials

### Debug Output
```
021/05/04 16:55:35 [WARN] Provider "registry.terraform.io/wgebis/mailgun" produced an invalid plan for mailgun_credential.example-1, but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
- .region: planned value cty.StringVal("us") does not match config value cty.NullVal(cty.String)

2021-05-05T08:56:35.509Z [WARN]  plugin.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
```

### Expected Behavior
No warning

### Terraform Configuration Files
```
resource "mailgun_route" "test-credential" {
    priority = "0"
    description = "inbound"
    expression = "match_recipient('.*@foo.example.com')"
    actions = [
        "forward('http://example.com/api/v1/foos/')",
        "stop()"
    ]
}
```

Regards,

**Comment by wgebis** [comment]
Thanks @emeric-martineau, it's addressed in #15. BTW: I invited you to the main plugin repo, so feel free to commit you changes to the feature branch 👍 .

**Comment by wgebis** [comment]
Fixed in v0.6.0
